redwood-layout-uiview/src/commonMain/kotlin/app/cash/redwood/layout/uiview/UIViewFlexContainer.kt

```diff
@@ -25,36 +25,36 @@ import app.cash.redwood.layout.widget.Row
 import app.cash.redwood.ui.Default
 import app.cash.redwood.ui.Density
 import app.cash.redwood.ui.Margin
+import app.cash.redwood.widget.ChangeListener
 import app.cash.redwood.widget.UIViewChildren
 import app.cash.redwood.yoga.AlignItems
 import app.cash.redwood.yoga.FlexDirection
 import app.cash.redwood.yoga.JustifyContent
 import platform.UIKit.UIView
 
 internal class UIViewFlexContainer(
-  direction: FlexDirection,
-) : Row<UIView>, Column<UIView> {
+  private val direction: FlexDirection,
+) : Row<UIView>, Column<UIView>, ChangeListener {
   private val yogaView = YogaUIView()
-
   override val value: UIView get() = yogaView
   override val children = UIViewChildren(value)
   override var modifier: Modifier = Modifier
 
   init {
     yogaView.rootNode.flexDirection = direction
     yogaView.applyModifier = { node, index ->
-      node.applyModifier(children.widgets[index].modifier, Density.Default)
+      if (index < children.widgets.count()) {
```

Comment:
Just curious--why is this change needed? How often is this called with `index >= children.widgets.count()`?

Reply:
i'll pull this out, this was an issue when I nested a LazyList in here and hard scrolled, but after rebasing, it's not crashing anymore